### ISSUE TITLE
When no renderer configuration is given, a more precise error would be helpful

### ISSUE THREAD

**Issue by samir-plusb** [honeybee/honeybee-agavi-cmf-vendor#23]
Sometimes when creating a view template, I just forget to add a corresponding renderer config. The error given then is: 

```
Type has no attribute: referenced_identifier
```

![error](https://cloud.githubusercontent.com/assets/4447755/17444719/cceda5a2-5b41-11e6-8b4b-0965a22b218e.png)

It is always the same error, but the cause is that the rendering config is missing and fixing this problem is very easy by just adding a renderer configuration. 

A more precise error message would be very handy here!

**Comment by Pictor13** [comment]
Yes, I noticed the same thing and was planning to throw a more meaningful message over there.
It would save from (indirectly) figuring it out, while configuring the views.

Thanks for reporting it!
